dawn/src/dawn/AST/Offsets.h

```diff
@@ -22,7 +22,8 @@
 #include <memory>
```

Comment:
Should the `#pragma once` be here in the header?

Reply:
Looks like the GitHub diff hid it. It's on line 15 above.